pkg/cli/admin/ocpcertificates/certregen/command.go

```diff
@@ -152,11 +153,20 @@ func (o *RegenerateCertificatesOptions) ToRuntime(args []string) (*RegenerateCer
 		return nil, err
 	}
 
+	dryRunStrategy, err := cmdutil.GetDryRunStrategy(cmd)
```

Comment:
standalone `--dry-run` is also deprecated. I hope that there is no script relying on passing `--dry-run` instead `--dry-run=server`.